src/vs/base/common/buffer.ts

```diff
@@ -66,12 +66,12 @@ export class VSBuffer {
 		this.byteLength = this.buffer.byteLength;
 	}
 
-	toString(): string {
+	toString(encoding: string = 'uft8'): string {
```

Comment:
I'm so stupid to let `utf8` be `uft8`.....

Reply:
@LeuisKen Why don't you just fix this typo and force push to your branch...